### PR TITLE
Caleb/scripts

### PR DESCRIPTION
**Pull request by caleb-kaiser**

### CODE REVIEW

**Comment by caleb-kaiser** [comment]
@dsblank when we upgrade Next and rewrite some of the "backend" logic, I'm going to refactor this such that the server automatically searches the `/scripts/` folder and injects the scripts it finds. But, for now, this works if you pass the names of the scripts you want injected as an env variable (e.g. `export SCRIPTS="google_analytics.js segment.js cracker_barrel.js"`), provided those filenames are present in the `/scripts/` folder

File: frontend/pages/_document.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Html, Head, Main, NextScript } from 'next/document';
-
+import Script from 'next/script';
+// TODO Remove this kangas.comet.com specific code when we migrate to Next 13
```
Comment (dsblank) [inline frontend/pages/_document.js:3]:
```suggestion
// TODO Refactor this hard-coded injection point when we migrate to Next 13
```

File: frontend/config.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const localConfig = {
     apiUrl: `${env.KANGAS_PROTOCOL || 'http'}://${env.KANGAS_HOST}:${env.KANGAS_BACKEND_PORT}/datagrid/`,
     defaultDecimalPrecision: 5,
     locale: 'en-US',
-    isColab: env.IN_COLAB === 'True'
+    isColab: env.IN_COLAB === 'True',
```
Comment (dsblank) [inline frontend/config.js:7]:
```suggestion
    isColab: env.IN_COLAB === 'True'
```

**Review by dsblank — APPROVED** [review]
Thanks!

**Pull request merged**